src/main/java/com/welovecoding/nbeditorconfig/options/EditorConfigOptionsPanel.java

```diff
@@ -17,6 +17,7 @@
    * WARNING: Do NOT modify this code. The content of this method is always
    * regenerated by the Form Editor.
    */
+  @SuppressWarnings("unchecked")
```

Comment:
The only way to fix it is to modify the code inside the method, which normally I would have done of course, but the code in question is the auto-generated code from using Netbeans' Form Designer. Unless you want to hardcode the Form and stop using the Designer for this class then there is very little to be done outside of just ignoring the warning since it is harmless. I hate ignoring warnings as well, but I didn't feel it was my place to just assume I could make such a big change and hard-code the form without the designer.